### ISSUE TITLE
Profiles for an arbitrary observable (or function of parameters)

### ISSUE THREAD

**Issue by ivborissov** [dkaschek/dMod#62]
Hello, 

`profile` can be used for parameters' identification or for constructing validation profiles (https://github.com/dkaschek/dMod/blob/34bc06d5be889da8e1830d4466144d7827dba307/vignettes/BA_transport_model.R#L248). Is there a way to use `profile` to construct confidence intervals for an arbitrary function of parameters or an observable at a given timepoint?

**Comment by marcusrosenblatt** [comment]
yes, you can specify an arbitrary time point in the datapointL2 function, and also an arbitrary target (name), this can also be any observable of the model. There is a technical issue sometimes. To run the profile there is at least one data point necessary for the particular observation. You could use a dummy data point with a high sigma value that does not contribute to the likelihood when this is necessary.

**Comment by ivborissov** [comment]
Marcus, thanks for your response! Maybe I have misunderstood you , but making sigma high in `datapointL2` results in almost zero contribution of `datapointL2` to the objective function. So an observable associated with this dummy point becomes non identifiable. And vise versa if I make sigma relatively small, validation bands become narrower and that seems to be a way to construct confidence bands

**Comment by marcusrosenblatt** [comment]
Hi Ivan,

My post was indeed maybe a bit hard to understand. I did not mean the sigma in the datapointL2 function. This sigma you should rather choose to be a small value as you said.

The arbitrary datapoint I meant would only be useful in the case that you do not have a particular observable or condition already in the model but want to calculate confidence intervals for this. Such as for example in the case of an arbitrary function of parameters as you asked for. You have to specify this function for example as an additional observable such that you are at first able to calculate a prediction for this with your model.
